### DISCUSSION TITLE
Pull request granularity

### DISCUSSION THREAD

**Discussion by jmdetloff** [biosimulations/biosimulations#4141] [Q&A] [answered]
A quick question about your preference IRT pull request granularity for the repo. Do you prefer these broken down to the feature level or the commit level? If I were to make some small formatting tweak or lint fix commits in the process of working on a feature, would you prefer these commits as part of that feature PR, all together in a single 'formatting' or 'linting' PR, or each in their own PR?

**Comment by jonrkarr** [comment, marked as answer]
We haven't been strict about this. Generally, we group functionally related changes together. Sometimes these have been quite large.

Grouping multiple formatting/linting changes into 1 PR seems fine to me.

Sometimes if we have large PRs that we want intermediate feedback on, we create draft PRs and then ask for feedback.
